apps/dashboard/config/configuration_singleton.rb

```diff
@@ -260,6 +260,11 @@ def pinned_apps
     config.fetch(:pinned_apps, [])
   end
 
+  # The length of the "Pinned Apps" navbar menu
+  def pinned_apps_menu_length
+    config.fetch(:pinned_apps_menu_length, 6)
```

Comment:
After we complete the other features (recent apps, favorite apps) we might play around with the idea default limits for the nav